app/models/category.rb

```diff
@@ -511,6 +514,16 @@ def self.find_by_slug(category_slug, parent_category_slug = nil)
   def subcategory_list_includes_topics?
     subcategory_list_style.end_with?("with_featured_topics")
   end
+
+  def trigger_category_created_event
+    DiscourseEvent.trigger(:category_created, self)
```

Comment:
Do we need this for uncategorized or default categories that are seeded?

Reply:
same as https://github.com/discourse/discourse/pull/5705#discussion_r177211381